### DISCUSSION TITLE
What is IpPort?

### DISCUSSION THREAD

**Discussion by Exergist** [jchristn/SuperSimpleTcp#59] [Q&A] [answered]
I'm uncertain what the `IpPort` property tied to `server.Events.ClientConnected` entirely means. Here is a snapshot from my simple server-client example (connecting via 127.0.0.1 at 12136):

![image](https://user-images.githubusercontent.com/31357974/110197867-f718ad00-7e13-11eb-8c34-25bdb4f47cff.png)

**The `127.0.0.1` portion is expected, but what does `50782` mean?**

For reference, here is my code for updating info on the server UI:
```csharp
private void Events_ClientConnected(object sender, ClientConnectedEventArgs e)
{
	this.Invoke((MethodInvoker)delegate
	{
		txtInfo.Text += $"{e.IpPort} connected.{Environment.NewLine}";
		lstClientIP.Items.Add(e.IpPort);
		btnSend.Enabled = true;
		btnDisconnect.Enabled = true;
	});
}
```

**Comment by jchristn** [comment, marked as answer]
Hi, that's the source TCP port.
